docs/sources/whatsnew/whats-new-in-v9-2.md

```diff
@@ -124,9 +124,9 @@ For details on using this functionality, see [GitHub pull request #55313](https:
 Transformations allow you to shape raw data from data sources, like metrics series or GitHub issues, so that you can visualize them as clearly as possible.
 We have extended the Join transformation to support both inner and outer JOINs, which work similarly to SQL inner and outer JOINs.
 
-Also, you can now click on the `x` to clear values in the select fields for the Outer Join and Grouping to Matrix transformations as expected.
+{{< figure src="/static/img/docs/transformations/transform-outer-join-9-2.png" max-width="750px" caption="Query builder groupings for Google Cloud monitoring" >}}
 
-Screenshot
+Also, you can now click on the `x` to clear values in the select fields for the Outer Join and Grouping to Matrix transformations as expected.
```

Comment:
Can we remove the comment here `Also, you can now click on the `x` to clear values in the select fields for the Outer Join and Grouping to Matrix transformations as expected.` as this is minor fix for a bug? I added it on the doc and got some feed back that we don't need it.